docs/specification/1.0.md

```diff
@@ -273,35 +272,37 @@ If a suggestion has no `uuid`, the EHR does not send a notification.
 
 If real-world performance were no issue, an EHR could launch a CDS Service
 passing only *context* data, and *without passing any additional clinical data*
-up-front. The CDS Service could then request any data it needed via the EHR's
+up-front. The CDS Service could then retrieve any data it needed via the EHR's
 FHIR REST API.
 
 But CDS services must respond quickly (on the order of 500 ms), and so we
 provide a performance tweak that allows a CDS Service to register a set of "prefetch
-templates" with the EHR ahead of time.
+templates" with the EHR within the context of the out-of-band Service Registration process.  
 
 The prefetch templates are a dictionary of `read` and `search` requests to supply
-relevant data, where the following variables are defined:
+the data minimally necessary to complete the requested decision-making process, where the following variables are defined:
 
 |variable|meaning|
 ---------|--------
 |`{{Patient.id}}`|The id of the patient in context for this activity (e.g. `123`)|
 |`{{User.id}}`|The type and id of the user for this session (e.g. `Practitioner/123`)|
 
-An EHR *may* choose to honor some or all of the desired prefetch templates from an appropriate source. For example:
+The CDS Hooks specification necessarily does not constrain how “prefetch” is implemented.  Whether and how prefetch is implemented is left up to the implementer, as architecturally dependent, and involves out-of-band negotiation between the health system, the EHR vendor, and the CDS service.  The prefetch MUST NOT exceed the user-based authorizations for the context from which the CDS service is called, and it MUST NOT exceed the data minimally necessary to complete the requested decision-making.  The EHR MAY  NOT prefetch any FHIR resources beyond those for which the current user is authorized.  
+
```

Comment:
Agree with the constraints around only providing data that the user is authorized to access, but disagree with the constraint that the EHR only provide data that is "minimally necessary" to complete the requested decision-making. That's a significant burden to place on the implementation and it would only apply to PHI, which is only that information that can actually be used to identify a patient.

Reply:
@bakerdb - The concern here is with the statement:

>and it MUST NOT exceed the data minimally necessary to complete the requested decision-making.

Was the intention to address [HIPAA's minimum necessary requirement](https://www.hhs.gov/hipaa/for-professionals/privacy/guidance/minimum-necessary-requirement/index.html)?

Or, was this statement trying to state that if a CDS Service wants the patient's current medications prefetched they should be given something unrelated like labs?

Bryn's concern (which I think is valid) is a scenario like this: Imagine a CDS Service needs just the medication name for its algorithm. It requests the patient's current medications to be prefetched. The EHR satisfies the prefetch request by sending the full FHIR medication resource.

Did the EHR violate the CDS Hooks standard by sending the full medication resource rather than just a paired down minimal medication resource composed of just the name? I would say no -- I have not seen discrete/granular control over individual data fields in systems I've worked on. As an aside, doing this detail level of fine-grain control over the fields is not possible with SMART on FHIR's scopes nor is it being implemented with FHIR implementations today in production.

Reply:
Because the data are provided within the context of a single patient, any data prefetched or available through an access token passed within that context are by definition "PHI."  However, I don't think the CDS Hooks specification should be a HIPAA-compliance mechanism.  So I have softened this language.